third_party/cpgfunctionEP-0.3.2/.gitignore

```diff
@@ -0,0 +1,7 @@
+cmake-build-debug/
```

Comment:
When viewing the files changed on GitHub, everything from here down is the new library. I'm not going to spend too much time in there reviewing, other than to note that most of these files are due to the library bringing in a stripped-down version of Eigen.